crates/linker/src/pe.rs

```diff
@@ -769,20 +1012,31 @@ mod test {
             r#"
             const std = @import("std");
 
-            extern fn magic() callconv(.C) u64;
+            extern const roc_one: u64;
+            extern const roc_three: u64;
+
+            extern fn roc_magic1() callconv(.C) u64;
+            extern fn roc_magic2() callconv(.C) u8;
 
             pub fn main() !void {
                 const stdout = std.io.getStdOut().writer();
-                try stdout.print("Hello, {}!\n", .{magic()});
+                try stdout.print("Hello, {} {} {} {}!\n", .{roc_magic1(), roc_magic2(), roc_one, roc_three});
```

Comment:
this is what we can do now: functions and constants, and more than 1 per section.